server/src/main/java/org/elasticsearch/common/bytes/ReleasableBytesReference.java

```diff
@@ -29,7 +31,7 @@
     private final BytesReference delegate;
     private final AbstractRefCounted refCounted;
 
-    public ReleasableBytesReference(BytesReference delegate, Releasable releasable) {
+    public ReleasableBytesReference(BytesReference delegate, Closeable releasable) {
```

Comment:
If `Page` were `Releasable` then we wouldn't need to do this I think.

Reply:
++ reverted all changes to this class now